eng/Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <ItemGroup>
+    <ProjectToBuild Include="$(RepoRoot)Experiment.sln" />
```

Comment:
NIT: I think you theoretically don't need this file as Arcade by default will build a solution on the root of the repo if it only finds one. At least that used to be the case.

Reply:
Correct: https://github.com/dotnet/arcade/blob/4f857c3c19e177cefdd85268151baf7bd8279422/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L73. I would just remove the whole file.